netcanv-renderer-opengl/src/font.rs

```diff
@@ -204,28 +229,36 @@ impl netcanv_renderer::Font for Font {
    }
 }
 
-pub(crate) struct GlyphRenderer<'face, 'store, 'gl> {
-   face: &'face Face,
+pub(crate) struct GlyphRenderer<'font, 'store, 'gl, 'c> {
```

Comment:
Darn, that's a lot of lifetimes... Out of curiosity, could you try consolidating them into one common lifetime `'a`? I've had that work just fine for me in the past.